kubetest2-gke/deployer/up.go

```diff
@@ -81,11 +81,19 @@ func (d *deployer) Up() error {
 					"--image-type="+image,
 					"--num-nodes="+strconv.Itoa(d.nodes),
 					"--network="+transformNetworkName(d.projects, d.network),
-					"--cluster-version="+d.Version,
 				)
 				if d.workloadIdentityEnabled {
 					args = append(args, fmt.Sprintf("--workload-pool=%s.svc.id.goog", project))
 				}
+				if d.ReleaseChannel != "" {
+					args = append(args, "--release-channel="+d.ReleaseChannel)
+					// --cluster-version must be a specific valid version if --release-channel is not empty.
+					if d.Version != "" && d.Version != "latest" {
+						args = append(args, "--cluster-version="+d.Version)
+					}
+				} else if d.Version != "" {
```

Comment:
the deployer itself doesn't allow empty string as the version because we want to be explicit about getting the latest version, and it's validated. so we can drop the empty string checks.

Reply:
Done. Thanks!